corehq/apps/sms/phonenumbers_helper.py

```diff
@@ -17,23 +17,44 @@ def parse_phone_number(number, region=None, failhard=True):
     from phonenumbers.phonenumberutil import parse as phonenumbers_parse, NumberParseException
     try:
         phone = phonenumbers_parse(number, region)
-        if phone:
-            return PhoneNumber(phone.country_code, phone.national_number)
     except NumberParseException:
         if failhard:
             raise PhoneNumberParseException()
         else:
             return None
 
+    return PhoneNumber(phone.country_code, phone.national_number)
+
 
 def strip_plus(number):
     return number[1:] if number.startswith('+') else number
 
 
-def get_country_code_and_national_number(number, failhard=False):
-    parsed = parse_phone_number(number, failhard=failhard)
-    if parsed:
-        country_code = parsed.country_code
-        national_number = strip_plus(number)[len(str(country_code)):]
-        return country_code, national_number
-    return None, None
+def get_country_code_and_national_number(number):
+    """
+    :param number: str representing phone number
+    :return: country_code: int, national_number: str
+    """
+    parsed = parse_phone_number(number, failhard=False)
+    return (parsed.country_code, str(parsed.national_number)) if parsed else (None, None)
+
+
+def country_code_for_region(region):
+    """
+    Simple wrapper around phonenumbers country_code_for_region method
+    :param region: str (examples: 'US', 'IN', 'ZA')
+    :return: int
```

Comment:
Does this always return an int, even if the country code was unknown?

Reply:
At first glance I thought yes. This is the method in phonenumbers:
```
def country_code_for_region(region_code):
    if not _is_valid_region_code(region_code):
        return 0
    return country_code_for_valid_region(region_code)
```
but if for some reason `_is_valid_region_code` has a bug, `country_code_for_valid_region` will raise an exception for an invalid code. Is that something we should plan for?

Reply:
We might not need to make any code changes, but the documentation should ideally note that it could raise an exception.

Reply:
See b1b46aab197c59f82208b34d0e0316e878740d95